test-app/tests/integration/components/radio-group-field-test.gts

```diff
@@ -119,6 +119,55 @@ module('Integration | Component | Fields | RadioGroup', function (hooks) {
     assert.dom('[data-radio-2]').isDisabled();
   });
 
+  test('it sets an individual radio to disabled with `@isDisabled`', async function (assert) {
+    await render(<template>
+      <RadioGroupField @label="Label" @name="group" data-group-field as |group|>
+        <group.RadioField
+          @label="option-1"
+          @value="option-1"
+          @isDisabled={{true}}
+          data-radio-1
+        />
+      </RadioGroupField>
+    </template>);
+
+    assert.dom('[data-radio-1]').isDisabled();
+  });
+
+  test('it sets an individual radio to readonly with `@isReadOnly`', async function (assert) {
+    await render(<template>
+      <RadioGroupField @label="Label" @name="group" data-group-field as |group|>
+        <group.RadioField
+          @label="option-1"
+          @value="option-1"
+          @isReadOnly={{true}}
+          data-radio-1
+        />
+      </RadioGroupField>
+    </template>);
+
+    assert.dom('[data-radio-1]').hasAttribute('readonly');
+  });
+
+  test('it sets readonly on the fieldset and all child radios using `@isReadOnly` at the root', async function (assert) {
+    await render(<template>
+      <RadioGroupField
+        @label="Label"
+        @name="group"
+        @isReadOnly={{true}}
```

Comment:
Thanks for adding this test! Seems like the DIL link is behind (at least when I'm checking it now) so it's nice to see tests that confirm this!